flor/__main__.py

```diff
@@ -62,10 +62,28 @@ def uninstall(multiuser=False):
         _, dst_root = f.read().strip().split(',')
 
     conda_flor_env = dst_root.split('/')[-1]
+
+    # Remove Flor script from user shell config file
+    shells_list = ['.zshrc', '.bashrc', '.cshrc', '.kshrc', '.config/fish/config.fish']
+    for s in shells_list:
+        shell_config = os.path.join(os.path.expanduser('~'), s)
+        if os.path.exists(shell_config):
+            import fileinput
+            file = fileinput.input(shell_config, inplace=True)
+
+            for line in file:
+                if 'flor() {' in line:
+                    for _ in range(5):
+                        # skip this line and next 5 lines
+                        next(file, None)
+                else:
+                    print(line.strip())
```

Comment:
Will `print` write to the shell config file? Is that what line 72 is for? Why did you choose not to open the file and just use a standard write?

Reply:
The `print` on line 80? Yes. I felt It's a cleaner to filter lines (remove the flor scripts) from the shell config files this way.

Reply:
do you have good reason not to use `write`?